server/tournaments/index.js

```diff
@@ -645,7 +644,7 @@ class Tournament {
 		}
 		if (this.autoDisqualifyTimer) clearTimeout(this.autoDisqualifyTimer);
 		const now = Date.now();
-		this.lastActionTimes.forEach((time, player) => {
+		for (const [player, time] of this.lastActionTimes) {
 			let availableMatches = false;
 			if (this.availableMatches.get(player).size) availableMatches = true;
 			const pendingChallenge = this.pendingChallenges.get(player);
```

Comment:
There's a lot of returns in the lines below this that need to be changed to `continue`

Reply:
(These are still not done)

Reply:
https://github.com/Zarel/Pokemon-Showdown/pull/5269/files#diff-9d3af10d83c5fb18f33b23271bd2d3f0R654, they're done or did I miss one

Reply:
You're right, they're done now.